lib/bots/versionbot.ts

```diff
@@ -1429,13 +1488,22 @@ export class VersionBot extends ProcBot {
 			_.each(protectedContexts, (proContext) => {
 				// We go through every status and see if the context prefixes the context
 				// of the status.
+				// We filter the VersionBot reviews.
 				_.each(statuses.statuses, (status) => {
 					if (_.startsWith(status.context, proContext)) {
-						// Did the check pass?
-						statusResults.push({
-							name: status.context,
-							state: statusLUT[status.state]
-						});
+						let includeContext = true;
+						// If there's a filter, determine if the result should be included or excluded.
+						if (filter) {
+							includeContext = filter.includeContexts ?
+								_.find(filter.contexts, (context) => context === status.context) !== undefined :
+								!_.find(filter.contexts, (context) => context === status.context);
```

Comment:
nit: Given that both branches of the ternary use `_.find(filter.contexts, (context) => context === status.context)`, I guess this code could be compacted slightly with the use of another variable? e.g.
```
const contextIsFiltered = _.find(filter.contexts, (context) => context === status.context);
includeContext = filter.includeContexts ? contextIsFiltered  !== undefined : !contextIsFiltered;
```
(and I'm not a TS programmer, but maybe that could be compacted further too)